src/components/modal/_modal.scss

```diff
@@ -142,3 +142,148 @@
     fill: $ui-05;
   }
 }
+
+@mixin modal--x {
+  .#{$prefix}--modal {
+    @include reset;
+    @include font-family;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: z('hidden');
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    content: '';
+    opacity: 0;
+    background-color: rgba($ui-05, 0.7);
+    transition: opacity 200ms, z-index 0s 200ms, visibility 0s 200ms;
+    visibility: hidden;
+
+    &.is-visible {
+      z-index: z('modal');
+      opacity: 1;
+      transition: opacity 200ms;
+      visibility: visible;
+    }
+  }
+
+  .#{$prefix}--modal--danger {
+    @include modal--color($support-01);
+  }
+
+  .#{$prefix}--modal-container {
+    @include reset;
+    @include layer('pop-out');
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-color: $ui-01;
+    width: 100%;
+    max-height: 100%;
+    height: 100%;
+
+    @include breakpoint(md) {
+      min-width: 500px;
+      max-width: 50%;
+      max-height: 90%;
+    }
+
+    @include breakpoint(lg) {
+      max-height: 80%;
+    }
+  }
+
+  .#{$prefix}--modal-header,
+  .#{$prefix}--modal-content {
+    width: 75%;
+    padding-left: 1rem;
+  }
+
+  .#{$prefix}--modal-header {
+    padding-top: 1rem;
+    margin-bottom: $spacing-xs;
+  }
+
+  .#{$prefix}--modal-header__label {
+    @include reset;
+    @include typescale('epsilon');
+    @include letter-spacing;
+    color: $text-02;
+    font-weight: 500;
+    margin-bottom: $spacing-2xs;
+  }
+
+  .#{$prefix}--modal-header__heading {
+    @include reset;
+    @include typescale('beta');
+    font-weight: 300;
+    color: $text-01;
+  }
+
+  .#{$prefix}--modal-content {
+    max-height: 83%;
+    overflow-y: auto;
+    margin-bottom: $spacing-3xl;
+    color: $text-01;
+    font-weight: 400;
+
+    > * {
+      @include reset;
```

Comment:
Wondering what this is for...?

Reply:
this is carried over from the old styles. I will look into whether or not we still need this mixin

Reply:
It looks like this mixin resets some default styles on the selected elements, but it seems we already have a global CSS reset. It looks like many of the mixins currently used can be removed, so I will refactor the styles to remove some dead CSS